.storybook/webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+module.exports = (storybookBaseConfig, configType, defaultConfig) => {
+    defaultConfig.module.rules.push(
+        {
+            test: /\.scss$/, use: ['style-loader', 'css-loader', 'sass-loader']
+        }
+    );
+
+    defaultConfig.resolve = {
+        alias: {
+            storybookHelpers: path.resolve(__dirname, '../src/__storybook_helpers__/')
+        },
+        extensions: ['.js']
+    };
+
+    return defaultConfig;
```

Comment:
indentation looks a bit off in this file, should be 2 spaces.

Reply:
fixed, thanks